requirements.txt

```diff
@@ -1,5 +1,6 @@
 # core deps
 boto==2.25.0
+boto3=1.3.1
```

Comment:
@cariaso This should be boto3==1.3.1. I think this will need to be added to setup.py to get install with pip.